model/application_authentication.go

```diff
@@ -15,7 +15,8 @@ type ApplicationAuthentication struct {
 	CreatedAt time.Time `json:"created_at"`
 	UpdatedAt time.Time `json:"updated_at"`
 
-	VaultPath string `json:"vault_path"`
+	// TODO: this doesn't exist yet
+	VaultPath string `json:"vault_path" gorm:"-"`
```

Comment:
`gorm: "-"` it doesn't create neither the column ?

Why we need `gorm:"-"` here ? I tough we are using it. Statuslistener's tests  started to fail(strange thing is that it is different error that here in CI) with this change.

Reply:
yea I think we can do it with just storing in Path.
thanks!